front/app/containers/Admin/pagesAndMenu/components/SectionFormWrapper.tsx

```diff
@@ -24,9 +24,11 @@ const SectionFormWrapper = ({
 }: Props) => {
   return (
     <>
-      <Box mb="16px" pb="40px">
-        {breadcrumbs && <Breadcrumbs breadcrumbs={breadcrumbs} />}
-      </Box>
+      {breadcrumbs && (
```

Comment:
Moved the condition outside of the Box wrapper, because to avoid 40px of empty padding-bottom when there are no breadcrumbs.

cc @lilkraftwerk